### PR TITLE
feat: Show placeholder message header for unavailable users

### PR DESCRIPTION
**Pull request by atomrc**
Will show a placeholder header when a user's metadata could not be fetched due to some federated backend being offline

image

### CODE REVIEW

File: src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx [hunk 1]
```diff
@@ -18,82 +18,111 @@
...
+      {sender.isDirectGuest() && (
+        <span
+          className="message-header-icon-guest with-tooltip with-tooltip--external"
+          data-tooltip={t('conversationGuestIndicator')}
+          data-uie-name="sender-guest"
+        >
+          <Icon.Guest />
+        </span>
+      )}
+    </>
```
Comment (tlebon) [inline src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx:80]: seems like these could all be combined into one function that returned the correct values from an object with a key that is equivalent to the sender?

Comment (V-Gira) [reply src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx:80]: something similar has been done here https://github.com/wireapp/wire-webapp/pull/14883 but not merged to dev yet. I think the idea is to come back and see if we can use the same reusable component later

Comment (atomrc) [reply src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx:80]: Yes, my plan is to use this one once it's merged. I'll have to identify if the contraints are really the same and go for it :)

Comment (tlebon) [reply src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx:80]: even #14883 seems a bit less DRY than id expect. the span component is duplicated several times.

Comment (tlebon) [reply src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx:80]: anyway thats fine then.

**Review by przemvs — APPROVED** [review]

**Pull request merged**